windows/src/global/delphi/general/KeymanPaths.pas

```diff
@@ -373,6 +375,24 @@ class function TKeymanPaths.KeymanConfigStaticHttpFilesPath(const filename: stri
   Result := Result + filename;
 end;
 
+class function TKeymanPaths.KeymanCustomisationPath: string;
+var
```

Comment:
As this isn't mentioned in the PR's description... it looks like the changes in this function also "throw in" a fix for ensuring the 'branded' version of the welcome page is presented correctly if one exists?

That's obviously somewhat related, but it's definitely a separate detail from the PR's title.

Reply:
Yeah, I should have mentioned this in the description sorry. This makes the search for the 'customisation' file (which is a really old term...) work properly for development machines as well as for installable builds. It's also consistent with other path searches.